.circleci/config.yml

```diff
@@ -731,7 +731,7 @@ jobs:
             conda activate ./env
             pushd docs
             pip install -r requirements.txt
-            make html
+            make html -SPHINXOPTS="-W"
```

Comment:
Maybe a better way would be to set

`SPHINXOPTS    = -W` in `doc/Makefile` so that contributors would also see the errors when they run `make html` locally?

Or we can just tell them to compile the docs by running `make html SPHINXOPTS="-W"`

Reply:
Thoughts on this @datumbox ? (It's marked as outdated but it's still relevant)

Reply:
I don't mind either, it's your call.